microcosm_logging/factories.py

```diff
@@ -57,6 +59,25 @@ def configure_logger(graph):
     return getLogger(graph.metadata.name)
 
 
+def enable_loggly(graph):
+    if graph.metadata.debug or graph.metadata.testing:
+        return False
+
+    if not graph.config.logging.loggly.enabled:
+        return False
+
+    try:
```

Comment:
Right. I really added the flag above to support explicitly *disabling* loggly. It's possible we don't need that flag now. I'll remove it.